### ISSUE TITLE
Brave flags `customizable` as misspelled, even though it has an entry in the Camdridge dictionary

### ISSUE THREAD

**Issue by hamirmahal** [brave/brave-browser#39603]
### Description

Brave flags `customizable` as misspelled, even though [`customizable` has an entry in the Camdridge dictionary](https://dictionary.cambridge.org/us/dictionary/english/customizable).

### Steps to reproduce

1. Go to any text area.
2. Type in
```
customizable
```

### Actual result

Brave flags `customizable` as misspelled.

### Expected result

Brave shouldn't have flagged `customizable` as misspelled.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

```
Brave	1.65.133 Chromium: 124.0.6367.208 (Official Build) (64-bit) 
Revision	a719189e6a0245a6ce7fab3ddc637e8bd75aed08
OS	Linux
```

### Channel information

- [ ] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [X] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by hamirmahal** [comment]
![image](https://github.com/brave/brave-browser/assets/43425812/97bec3a9-27cd-43f3-ae6f-e0e1358328df)

**Comment by hamirmahal** [comment]
I saw the `Add to dictionary` option beforehand; I assume users wouldn't want words like these to be flagged by Brave.

How would one contribute a dictionary update? Is that a change one would make to `brave-browser`, or would it have to be an upstream change in `chromium`?

**Comment by rebron** [comment]
@hamirmahal Yes. We'll wait for an upstream change in chromium for this one. There's likely an issue already filed there.
